jbpm-services/jbpm-services-api/pom.xml

```diff
@@ -6,6 +6,7 @@
     <version>7.18.0-SNAPSHOT</version>
   </parent>
   <artifactId>jbpm-services-api</artifactId>
+  <version>7.17.0-SNAPSHOT</version>
```

Comment:
I believe that the build failed with a compilation error because this version was changed to the older one. It shouldn't be here at all.

Reply:
Sorry! It was an error due to the rebase. Fixed.